### DISCUSSION TITLE
なんUに相当するエッヂUの実装提案

### DISCUSSION THREAD

**Discussion by edgnu0011** [edginer/eddiner#12] [Ideas]
エッヂの複数板対応とデーターベース分割が完了し過去ログのdat取得も本格運用に入ったようですので、最初の複板実装としてなんUのエッヂバージョン: エッヂUを検討いただけないでしょうか。
以下3点において最初の複板実装として運用の見通しが良いと思われます。

1. 基本的になんJ文化を継承しているスレッドと住民であるため現住のエッヂ民との齟齬が少ない
2. 通報が必要になるようなレベルのトラブルを起こすスレッドがなく、高速で実況されるスレッドもない
3. エッヂでは板更新が高速なため落ちてしまう、すこしゆるい速度(500-1000レス/日)の話題のスレッドをエッヂUでカバーすることが可能となる

またなんU側の事情では板全体に対するスクリプトによる埋め立てが再開し3chan.cc/liveuranus/の見通しが不明な中、避難または移転場所を検討しており、
下記のような条件で掲示板を精査したところエッヂしか候補が残らない状況なため本件の提案をさせて頂いた次第です。
1. 要認証書き込み
2. DDoS対応
3. 過去ログのdat落ちと専ブラによる過去ログのdat取得
4. 管理人が技術者

以上よろしくお願いいたします。

**Comment by edginer** [comment]
当面の間はリスクやコストの観点から新規に板を作成する予定はありません
ただ、将来的には作成する可能性があるためその場合に再度検討させていただきます
